src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -297,6 +333,14 @@ export default class ClientMetricReport {
     },
     frameHeight: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_DECODE_HEIGHT },
     frameWidth: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_DECODE_WIDTH },
+    totalDecodeTime: {
```

Comment:
should this name be changed?

Reply:
This key is just about the field name in webrtc report where the metric is derived from, we cannot change that naming as it comes from webrtc..